sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -255,12 +300,21 @@
       </item>
       <item>
           <EFDB_Name>rainLevel</EFDB_Name>
-	  <Description>Rain Level Telemetry</Description>
+	      <Description>Rain Level Telemetry</Description>
           <IDL_Type>float</IDL_Type>
           <Units>mm</Units>
           <Conversion></Conversion>
           <Sensor_location></Sensor_location>
           <Count>1</Count>
       </item>
+      <item>

```

Comment:
Please move that as a first item. Seems as consensus among subsystems I know (M1M3,MTMount) - timestamp is the key and therefore the most important (kind of) part of the message, and should be on top.

Reply:
Done.